AUTHORS

```diff
@@ -2683,5 +2683,170 @@ Michael Wei <mwei@cs.ucsd.edu>
+Alex Aubuchon <alex@aub.dev>
+Samuel Attard <samuel.r.attard@gmail.com>
+rexagod <rexagod@gmail.com>
+Antonio Kukas <tonykukas@gmail.com>
+murgatroid99 <mlumish@google.com>
+Saagar Jha <saagar@saagarjha.com>
+vmarchaud <contact@vmarchaud.fr>
+Milad Farazmand <miladfar@ca.ibm.com>
+mutao <mutao-hf@loongson.cn>
+Samantha Sample <ssample812@gmail.com>
+nicolasrestrepo <nicolasrestrepo34@gmail.com>
+Angie M. Delgado <amelisdl@gmail.com>
+Alex Ramirez <alexander.ramirez@gmail.com>
+Duvan Monsalve <duvanmonsa@gmail.com>
+Luis Gallon <luisgallon@gmail.com>
+kball <kball@zendev.com>
+MistyBlunch <gracenikole@gmail.com>
+Laura Ciro <ltciro@gmail.com>
+Yomar <yomar.guti@gmail.com>
+raveneyex <raveneyex@gmail.com>
+khriztianmoreno <khriztianmoreno@gmail.com>
+David Sánchez <d4vsanchez@gmail.com>
+melinamejia95 <melinamejia95@gmail.com>
+David Carlier <devnexen@gmail.com>
+Benoît Zugmeyer <bzugmeyer@gmail.com>
+Julian Correa <julian.alexis.correa@gmail.com>
+Felipe <afvasquezt@gmail.com>
+Juan Roa <jdroa92@gmail.com>
+Ivan Villa <trezeguet55@gmail.com>
+Caleb ツ Everett <calebev@amazon.com>
+Miken <omarlozano053@gmail.com>
+Eugene Ostroukhov <eostroukhov@gmail.com>
+Gabriela Niño <gabynr@gmail.com>
+Mike MacCana <mike.maccana@gmail.com>
+Tim Baverstock <tim.baverstock@corp.badoo.com>
+Walle Cyril <cyril.walle@protonmail.com>
+Xu Meng <mengxumx@cn.ibm.com>
+Samuel Attard <sattard@slack-corp.com>
+Ben L. Titzer <titzer@google.com>
+Ojasvi Monga <ojasvi@Ojasvis-MacBook-Air.local>
+Shajan Jacob <shajanjp@gmail.com>
+Austin Wright <aaa@bzfx.net>
+Vickodev <harvic3@gmail.com>
+Karen He <32376376+baekrxnn@users.noreply.github.com>
+Harshitha KP <harshi46@in.ibm.com>
+Tanner Stirrat <tstirrat@gmail.com>
+h3knix <h3knix@gmail.com>
+Cotton Hou <himcotton@gmail.com>
+Edward Vielmetti <edward.vielmetti@gmail.com>
+Micha Hanselmann <deermichel@github.com>
+Luca Lindhorst <info@lucalindhorst.de>
+Manuel Ochoa Loaiza <mochoa1127@gmail.com>
+Juan Bedoya <juansb827@gmail.com>
+Andres Bedoya <mortiis.angel@gmail.com>
+elyalvarado <elyalvarado@gmail.com>
+Felipe Duitama <felipedc09@gmail.com>
+Alejandro Nanez <alejonanez@gmail.com>
+Jeroen Ooms <jeroenooms@gmail.com>
+PaulBags <19583196+PaulBags@users.noreply.github.com>
+EduardoRFS <theeduardorfs@gmail.com>
+Natalie Fearnley <nfearnley@gmail.com>
+pi1024e <doremylover123@gmail.com>
+Giorgos Ntemiris <ntemirisgiorgos3@gmail.com>
+Rainer Poisel <rainer.poisel@gmail.com>
+Andrew Hughes <Andrew.Hughes1@ibm.com>
+Tony Brix <tony@brix.ninja>
+Anas Aboureada <anas.ieee@gmail.com>
+MattIPv4 <matthew@cowley.org.uk>
+David Guttman <david@js.la>
+Xavier Stouder <xavier@stouder.io>
+ran <abbshrsoufii@gmail.com>
+Nick Schonning <nschonni@gmail.com>
+Chetan Karande <kchetan.tech@gmail.com>
+Bradley Farias <bfarias@godaddy.com>
+Nimit Aggarwal <nimitagg95@gmail.com>
+Devendra Satram <devendra4sci@gmail.com>
+AtticusYang <yyongtai@163.com>
+Kamil Rytarowski <n54@gmx.com>
+zhangyongsheng <zhangyongsheng@youzan.com>
```

Comment:
Aha, this is me, but can I still use the original one ?https://github.com/nodejs/node/blob/ac4195992290ef6f0b62e2b218ee0dba059f2a76/.mailmap#L419

Reply:
@ZYSzys Sure, done!

Reply:
Thank you!